implementations/code_df/conditions.py

```diff
@@ -254,3 +254,30 @@ def set_commits(self, commits):
         df = df[~df['merge']]
 
         self.included = set(df['hash'].tolist())
+
+
+class CommitByTag(Commit):
+    """
+    Considerer as a commit contains a specific tag into its message.
+    """
+
+    def __init__(self, tag):
+        self.tag = tag
+
+    def set_commits(self, commits):
+        """
+        Check if a commit contains the tag in this message.
+
+        Exemple: we want to have the commits when the message begins by [perceval] or [backend]
+
```

Comment:
I think the `set_commits` docstring can be improved to be like the other sub classes of `Commit`.